services/alert-stream-broker/values.yaml

```
@@ -1 +1,6 @@
-# This file intentionally blank - no customization needed
+strimzi-registry-operator:
+  # Should match the cluster name used by the alert-stream-broker
+  clusterName: alert-broker
+
+  # Should match the namespace where the alert-broker cluster runs
+  watchNamespace: alert-stream-broker
```

This is missing some necessary configurations. See the [sasquatch](https://github.com/lsst-sqre/phalanx/blob/3d565962e3c300a643dd163c9a72c906d05169d9/services/sasquatch/values.yaml#L7-L10) for guidance.

```suggestion
strimzi-registry-operator:
  # Should match the cluster name used by the alert-stream-broker
  clusterName: alert-broker

  # Should match the namespace where the alert-broker cluster runs
  watchNamespace: alert-stream-broker
  
  operatorNamespace: "alert-stream-broker"
```